README.md

```diff
@@ -65,7 +65,7 @@ For more details about the output files and reports, please refer to the
 
 ## Credits
 
-qbic-pipelines/vcftomaf was originally written by [SusiJo](https://github.com/SusiJo) and [Friederike Hanssen](https://github.com/FriederikeHanssen)
+qbic-pipelines/vcftomaf was originally written by [SusiJo](https://github.com/SusiJo), [Friederike Hanssen](https://github.com/FriederikeHanssen) and [Famke Bäuerle](https://github.com/famosab).
```

Comment:
I'm not sure to what extend or since when you are involved. But assuming Susanne & Rike made it to the first release and you are continuing the pipeline, then I'd recommend something like `originally written by Susanne & Rike. The pipeline was developed by Famke after release 1.0.0`.